src/test/java/lithium/university/TwitterRetrieveTest.java

```diff
@@ -0,0 +1,64 @@
+package lithium.university;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import twitter4j.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TwitterRetrieveTest {
+    @Mock
+    private Twitter twitterTest;
+
+    @InjectMocks
+    private TwitterRetrieve twitterRetrieveTest;
+
+    private Status mockStatus() {
+        return Mockito.mock(Status.class);
+    }
+
+    @Test(expected = NegativeArraySizeException.class)
+    public void testRetrieveNegative() throws TwitterException {
+        List<Status> l = twitterRetrieveTest.retrieveFromTwitter(twitterTest, -1);
+        System.out.println("testRetrieveNegative pass");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testRetrieveNothing() throws TwitterException {
+        List<Status> l = twitterRetrieveTest.retrieveFromTwitter(twitterTest, 0);
+        System.out.println("testRetrieveNothing pass");
+    }
+
+    @Test
+    public void testRetrieveSomething() throws TwitterException {
+        ResponseList<Status> fakeList = new FakeResponseList<>();
+        fakeList.add(mockStatus());
+        Mockito.when(twitterTest.getHomeTimeline(Mockito.any(Paging.class))).thenReturn(fakeList);
+        List<Status> l = twitterRetrieveTest.retrieveFromTwitter(twitterTest, 1);
+        System.out.println("Size Expected: 1, Size Actual: " + l.size());
+        if(l.size() == 1){
```

Comment:
I'd recommend a bit more checking as opposed to just the size to verify it's not going to twitter.  It's possible twitter just returned back 1 item and the test would pass right?